src/View/Helper/PaginatorHelper.php

```diff
@@ -1106,4 +1153,21 @@ public function implementedEvents()
     {
         return [];
     }
+
+    /**
+     * Helper method to deprecate the 'model' option throughout PaginationHelper.
+     *
+     * @deprecated
+     * @param array $options Array of options
+     * @return array Array of options
+     */
+    protected function _deprecateModelOption(array $options)
+    {
+        if (isset($options['model'])) {
+            $options['scope'] = $options['model'];
```

Comment:
Say someone has written code like:

```php
$this->Paginator->options(['model' => 'Client', 'scope' => 'ThinClient']);
```

... having overloaded Paginator with his own PaginatorHelper which extends upon the core PaginatorHelper.
Here the meaning of scope is/was a different.

I can change it to B, if you say so. I am just not sure it is entirely better.